### ISSUE TITLE
Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by xinqiangSong** [foxitsoftware/react-native-foxitpdf#3]

**Comment by xinqiangSong** [comment]
You can solve this problem through the following links：

https://stackoverflow.com/questions/48756550/unable-to-resolve-module-accessibilityinfo-when-trying-to-create-release-bund
